cmd/image/qcow2ova/get-image.go

```diff
@@ -0,0 +1,85 @@
+package qcow2ova
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+
+	"k8s.io/klog/v2"
+)
+
+// Downloads or copy the image into the target dir mentioned
+func getImage(dir string, src string) (string, error) {
```

Comment:
Can the variable names be more descriptive? (srcUrl, downloadDir) ?? Not mandatory, but kind of helps later with understanding